scripts/Dockerfile

```
@@ -5,8 +5,16 @@ MAINTAINER chriseth <chris@ethereum.org>
 #Establish working directory as solidity
 WORKDIR /solidity
 
-# Build dependencies
-RUN apk update && apk add boost-dev boost-static build-base cmake git clang
+# Build dependencies 
+RUN apk update && \
+    apk add --no-cache --no-install-recommends \
+        boost-dev \
+        boost-static \
+        build-base \
+        cmake \
+        git \
+        clang && \
+    rm -rf /var/cache/apk/*
```

If we're at refactoring the Dockerfiles, I'll just add that I'd prefer if we eventually switched to this style for multi-command `RUN`:
```suggestion
RUN <<-EOF
    set -eou pipefail
    apk update
    apk add --no-install-recommends \
        boost-dev \
        boost-static \
        build-base \
        cmake \
        git \
        clang
    rm -rf /var/cache/apk/*
EOF
```
It has less noise from continuations and `&&` and does not require changing the snippet when it is moved to/from an `.sh` file.